lib/aruba/api.rb

```diff
@@ -729,6 +735,61 @@ def io_wait
       @aruba_io_wait_seconds || DEFAULT_IO_WAIT_SECONDS
     end
 
+    DEFAULT_ROOT_DIRECTORY = Dir.getwd
+
+    # The root directory of aruba
+    def root_directory
+      @aruba__root_directory ||= DEFAULT_ROOT_DIRECTORY
+    end
+
+    DEFAULT_FIXTURES_PATH_PREFIX = '%/'
+
+    def fixtures_path_prefix
+      @aruba_fixtures_path_prefix || DEFAULT_FIXTURES_PATH_PREFIX
+    end
+
+    DEFAULT_FIXTURES_DIRECTORY = 'fixtures'
+
+    # The path to the directory which contains fixtures
+    # You might want to overwrite this method to place your data else where.
+    #
+    # @return [String]
+    #   The directory to where your fixtures are stored
+    def fixtures_directory
```

Comment:
I think the finding of the fixture directory can be implemented such that it is only executed once only. I.e. Setting a class variable straight in the class body. And making it configurable like `#exit_timeout` and `#io_wait` Does it make sense?